vpdq/README.md

```diff
@@ -12,16 +12,16 @@ Both TMK and vPDQ are backed by PDQ, and so inherit both PDQ’s strengths and w
 ## Producing a Hash
 The algorithm for producing the “hash” is simple: given a video, convert it into a sequence of frame images at some interval (for example, 1 frame/second). For each frame image, use the PDQ hashing algorithm on each.
 
-We can annotate these hashes with their frame number. So for a 5 minute video at 1 frame/sec, we might have:
-| Frame | PDQ Hash|
-| ------------- | ------------- |
-| 1  | face000...  |
-| 2  | face000...  |
-| 3  | face011...  |
-| 4  | face111...  |
-| ... | ...  |
-| 29999 | 88784444 |
-| 30000 | 88884444 |
+We can annotate these hashes with their frame number, quality(0-100 which measures gradients/features,from least featureful to most featureful) and time stamp(sec). So for a 5 minute video at 1 frame/sec, we might have:
+| Frame | Quality | PDQ Hash | Time Stamp |
```

Comment:
(nit): I usually see timestamp is one word: https://www.computerhope.com/jargon/t/timestam.htm#write

(unsure) You may want to clarify somewhere that this is seconds time, but I'm not sure if timestamp is well understood in this space.